<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Admin</class>
 <widget class="QWidget" name="Admin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>20</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>18</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>ADMIN MENU</string>
   </property>
  </widget>
  <widget class="QListView" name="listDogs">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>781</width>
     <height>251</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <bold>true</bold>
    </font>
   </property>
   <property name="inputMethodHints">
    <set>Qt::InputMethodHint::ImhNoEditMenu</set>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
   </property>
   <property name="itemAlignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>340</y>
     <width>591</width>
     <height>151</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="statistics">
      <property name="text">
       <string>STATISTICS</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="addDog">
      <property name="text">
       <string>ADD DOG</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="deleteDog">
      <property name="text">
       <string>DELETE DOG</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="UpdateDog">
      <property name="text">
       <string>UPDATE DOG</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="logout">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>540</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>LOGOUT</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
